capitoli/Sesto_Passo_Istruzioni_Condizionali.rst

```diff
@@ -44,12 +51,504 @@ prima la parola if, poi la condizione poi i : (due punti), e
+
+**L’istruzione deve stare più all’interno della prima riga** (hai presente i 
+margini del foglio? Se la prima riga è allineata al margine sinistro, le linee 
+successive devono stare più a destra). Si dice che devono essere “indentate”. 
+
+Dopo la “condizione” anziché una sola istruzione, come negli esempi 
+precedenti, possono essere scritte due o più istruzioni:
+
++--------------------------+-------------------------+
+|struttura                 |:blue:`esempio`          |
++--------------------------+-------------------------+
+| if<condizione>:          | if voto>=6:             |
++--------------------------+-------------------------+
+|  <istruzione1>           |  print("promosso")      |
++--------------------------+-------------------------+
+|  <istruzione2>           | print("mamma contenta") |
++--------------------------+-------------------------+
+
+Non c'è un limite al numero di istruzioni che possono comparire nel corpo di 
+un'istruzione if ma deve sempre essercene almeno una.	
+
+**La prima riga di istruzioni che non sta più all'interno (che non e' più** 
+**“indentata”) segnala al computer la fine del blocco di istruzioni e non**
+**ne fa parte.** 
+
+Prova a scrivere questo programma:
+**if 5 > 10**
+
+      **print ("sun")**
+ 
+**print ("moon")** 
+
+il computer scriverà “moon” perchè la linea non fa più parte dell’istruzione if. 
+Ma se la linea print (“moon”) viene indentata farà parte dell’if e non verrà più 
+stampata. La versione corretta del programma è:
+
+:blue:`if 5 > 10:`
+
+     :blue:`print ("sun")`
+
+     :blue:`print ("moon")`
+
+Vediamo adesso un esempio in cui e' un po' più 
+complicato prendere una decisione perché 
+consideriamo più condizioni insieme:
+
+Hai più di sette anni?
+Sei bravo a giocare a calcio?
+Se sì puoi iscriverti alla scuola calcio del Milan.
+
+:red:`ragazzo = input (“Hai più di sette anni?”)`
+
+:red:`campione = input (“Sei bravo a giocare a calcio?”)`
+
+:red:`if ragazzo == “sì”:`
+
+    :red:`if campione == “sì”:`
+
+         :red:`print (“Puoi iscriverti alla scuola calcio del Milan”)`    
+
+
+Per semplificare la scrittura del programma, possiamo usare tre nuovi operatori 
+chiamati :boltblue:`or, and, not:`
+
++-------------------+---------------------+
+|     opertaore     |    significato      | 
++===================+=====================+
+| or                | "oppure"            |
++-------------------+---------------------+
+| and               | "e inoltre"         |
++-------------------+---------------------+
+| not               | "non"               |
++-------------------+---------------------+
+
+Usando gli operatori logici, l’ultimo programma che abbiamo scritto diventa 
+molto semplice:
+
+:red:`if vecchietto == ”sì”` :blue:`and` :red:`campione == ”sì”:`
+
+      :red:`print (“Puoi iscriverti alla scuola calcio del Milan”)`
+
+Vediamo alcuni esempi concreti in cui questi nuovi operatori sono necessari. 
+
+
+
+:blue:`And`
```

Comment:
Visto che è un titolo bisogna dargli un nome più significativo. Direi "Operatore Logico AND"